.github/workflows/modules_and_generate.yml

```
@@ -48,5 +48,5 @@ jobs:
       - name: go mod tidy
         run: |
           go generate ./...
-          go mod tidy
+          go mod tidy -go=1.16 && go mod tidy -go=1.17
```

Maybe we can just use
```suggestion
          make tidy
```